api/v1alpha1/datamigrate_types.go

```diff
@@ -88,6 +88,21 @@ type DataMigrateSpec struct {
 	// Resources that will be requested by the DataMigrate job. <br>
 	// +optional
 	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
+
+	// Parallelism defines the parallelism tasks
+	// +optional
+	// +kubebuilder:default:=1
+	// +kubebuilder:validation:Minimum=1
+	Parallelism int32 `json:"parallelism,omitempty"`
+
+	// SSHSecretName defines the secret name used for ssh. It must be set when Parallelism is greater than 1.
+	// +optional
+	SSHSecretName string `json:"sshSecretName,omitempty"`
```

Comment:
Same as `WorkersReadyTimeout`, how about move `SSHSecretName` to `DataMigrate.spec.options`?

Reply:
I agree with moving `WorkersReadyTimeout` to` DataMigrate.spec.options`, but the `SSHSecretName`  is not optional when Parallelism is greater than 1.